tests/corrupted_stamp_test.cpp

```diff
@@ -125,6 +124,7 @@ namespace tests {
 void test_random_valid_headers(unsigned int number) {
 	std::default_random_engine random_generator;
 
+	std::discrete_distribution<uint8_t> version_distribution({10, 90}); // version=0: 10%, version=1: 90%
```

Comment:
Why is it such relation (10% vs 90%)?

Reply:
I think that uniform distribution for version doesn't behave as true. I don't insist.

Reply:
Why should the test have this distribution?

Reply:
Test cases are random generated. There's no guarantee that they each time will be generated to cover all the functional cases. False-positive test pass can occur. The main cases must be tested more properly, because false-positive result is more critical for them.